docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,872 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Prebuilt {anomaly-jobs}
+
+{anomaly-jobs-cap} contain the configuration information and metadata necessary 
+to perform an analytics task. {kib} can recognize certain types of data and 
+provide specialized wizards for that context. This page lists the available 
+prebuilt {anomaly-jobs} that you can use via {kib}. Every module item in the 
+list contains a name, a short description, information about 
+the {ml-docs}/ml-buckets.html#ml-buckets[`bucket_span`] and the 
+{ml-docs}/ml-functions.html[analysis function] as well as the influencers – the 
+entities that have contributed to, or are to blame for, the anomalies – of the 
+{anomaly-detect} job.
+
+
+[float]
+[[ootb-ml-jobs-apache]]
+=== Apache
+
```

Comment:
I think it might be worthwhile to add tagged sections here so that the content can be potentially re-used in other books:
```suggestion
tag::apache-jobs[]
```